File: python/dateTime.py
# Python program showing how Python's built-in datetime module works

from datetime import datetime

dt = datetime.now() # Current date and time

# Date
day = dt.day # Current day
month = dt.month # Current month
year = dt.year # Current year

# Hour
hour = dt.hour # Current hour
minute = dt.minute # Current minute
second = dt.second # Current second
microsecond = dt.microsecond # Current microsecond

print(f"Current date: {day}/{month}/{year}")
print(f"Current hour: {hour}:{minute}:{second}")

File: js/dateTime.js
// program to display the date and time
// get date and time
const date = new Date(2017, 2, 12, 9, 25, 30);

// get the date as a string
const n = date.toDateString();

// get the time as a string
const time = date.toLocaleTimeString();

// display date
console.log('Date: ' + n);

// display time
console.log('Time: ' + time);
